### PR TITLE
[mercadopago-sdk-js] Added new value props for Wallet Brick (credits and smart) 

### PR DESCRIPTION
**Pull request by icaldana**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [here](https://www.mercadopago.com.br/developers/en/docs/checkout-bricks/wallet-brick/additional-customization/change-texts)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by francineemilia — APPROVED** [review]

**Review by meliguicarvalho — APPROVED** [review]

File: types/mercadopago-sdk-js/mercadopago-sdk-js-tests.ts [hunk 1]
```diff
@@ -32,6 +32,25 @@ brickBuilder.create('cardPayment', 'container', {
     },
 });
 
+brickBuilder.create('wallet', 'container', {
```
Comment (andreagostinho-meli) [inline types/mercadopago-sdk-js/mercadopago-sdk-js-tests.ts:35]: Não teria que passar uma preference já que a value prop é `smart_option`?

Comment (icaldana) [reply types/mercadopago-sdk-js/mercadopago-sdk-js-tests.ts:35]: Not for the typing purposes, only when really implementing it.

**Review by ghost — APPROVED** [review]

**Comment by icaldana** [comment]
ready to merge

**Comment by icaldana** [comment]
Ready to merge

**Comment by icaldana** [comment]
> @icaldana: Everything looks good here. I am ready to merge this PR (at [bf1c7e0](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/bf1c7e0019d79df615ede914e065f6f47bfe809f)) on your behalf whenever you think it's ready.
> 
> If you'd like that to happen, please post a comment saying:
> 
> > Ready to merge
> 
> and I'll merge this PR almost instantly. Thanks for helping out! ❤️
> 
> (@mercadopago: you can do this too.)

Ready to merge

**Review by ghost — APPROVED** [review]

**Review by francineemilia — APPROVED** [review]

**Comment by francineemilia** [comment]
ready to merge

**Comment by icaldana** [comment]
Ready to merge

**Pull request merged**